python/lsst/pipe/tasks/subtractBrightStars.py

```diff
@@ -266,6 +266,9 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         if self.config.doWriteSubtractedExposure:
             outputExposure = inputs["inputExposure"].clone()
             outputExposure.image -= subtractor.image
+            if self.config.doApplySkyCorr and (inputs['skyCorr'] is not None):
+                # adding the skyCorr frame back to the subtracted calexp
```

Comment:
I try to leave out in-line comments that just describe the code verbatim, and I think L271 below is fairly self-explanatory. I'll leave it up to you if you want to leave this in or not.